book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,179 @@
+# Simple Payment and State Verification
+
+The transaction contains a signature. Validators store those signatures for a
+short period of time to ensure the transaction is not processed more than once.
+Validators provide a JSON RPC endpoint, which clients can use to query the
+cluster if a transaction has been recently processed. Validators also provide a
+PubSub notification, whereby a client registers to be notified when a given
+signature is observed by the validator. While these two mechanisms allow a
+client to verify a payment, they are not a proof and rely on a trusted
+validator.
+
+## Light Clients
+
+Light clients are cluster participants that are not running a fullnode.  The
+goal of this proposal is to provide a level of security that is greater than
+trusting a remote fullnode without requiring the light client to spend a lot of
+resources on verifying the ledger.
+
```

Comment:
I expected a paragraph here starting with something like: "A light client provides proofs by..." or some such description of how a light client provides that level of security.

Reply:
I've added the following at the beginning of the next paragraph

```
Rather than providing transaction signatures directly to a light client, the
fullnode instead generates a Merkle Proof from the transaction of interest to
the root of a Merkle Tree of all transactions in the including block. This Merkle
Root is stored in a ledger entry which is voted on by validators, providing it
concensus legitimacy.  ...
```